modules/rostests/apitests/ntdll/NtRaiseHardError.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for NtRaiseHardError
```

Comment:
Is this actually related to jobs or does it just happen to be here?

Reply:
This is how I came up with the change in dll/win32/kernel32/client/except.c
Initially I thought that the job object would actually set the flag (as hinted by msdn).